### PR TITLE
Issue #552:Add functionality for comment button at top of blogs

### PR DESCRIPTION
**Pull request by flowersw**
@kostajh , with your and @AnneTee 's help, this added the functionality we wanted. Though I will continue to push to this branch for the new fix we need to add.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -178,3 +87,108 @@ $(document).ready(
                 });
             });
     });
+
+// Export comment functionality and include functionality for comment count button
+function commentFunction() {
+            $('#js-expander-trigger').toggleClass("expander-hidden");
```
Comment (AnneTee) [inline assets/js/main.js]: @flowersw This line should be indented 4 spaces, not 8

File: assets/js/main.js [hunk 2]
```diff
@@ -178,3 +87,108 @@ $(document).ready(
...
+
+
+    // Create variable for request URI.
+            var commentServer = "{{ site.comment_server_url }}";
```
Comment (AnneTee) [inline assets/js/main.js]: @flowersw Same thing, this line and the lines below are indented too many spaces

File: assets/js/main.js [hunk 3]
```diff
@@ -178,3 +87,108 @@ $(document).ready(
                 });
             });
     });
+
+// Export comment functionality and include functionality for comment count button
+function commentFunction() {
```
Comment (AnneTee) [inline assets/js/main.js]: @flowersw This is nitpicky, but I wouldn't include the word "function" in a function name. You might call it something like `enableCommentForm()` or something else that describes what it does a little more

**Comment by flowersw** [comment]
@AnneTee , with @dmurphy1 's help, this latest commit (f67f613) add's the functionality we're looking for, and solved the toggling class problem we ran into. Can you think of a more optimal solution?

**Comment by AnneTee** [comment]
@flowersw Nice, I tested this and it works well!

**Comment by flowersw** [comment]
Ok @AnneTee , changes have implemented. In this latest commit, I also got rid of several other unused variables, and a few pieces of redundant code. Manually testing, everything seemed fine. Should we implement more tests/create a different issue/pr for this kind of clean-up? If not, I think this is ready to go. Thanks

**Comment by AnneTee** [comment]
@flowersw No, I think it's fine to include it here

I do think @kostajh should review all this though.

**Comment by chrisarusso** [comment]
I'm merging and closing for now. @kostajh fine for refinements, but works in my testing.

**Pull request merged**